packages/@lwc/integration-karma/test/rendering/dynamic-slots/x/parent/parent.js

```diff
@@ -0,0 +1,30 @@
+import { LightningElement, api } from 'lwc';
+
+export default class Parent extends LightningElement {
+    undefinedName;
+    named1Name = 'named1';
+    slotWithDefaultContentName = 'named-with-default-content';
+    counter = 1;
+    number = 1;
+    boolean = true;
```

Comment:
Add a case for boolean value `false` and have a slot named `"false"` in the child. I expect that content to be slotted into the `false` slot and not the default slot.